src/workflows/import-content.ts

```diff
@@ -50,8 +50,7 @@ export type ImportContentOutput = {
 };
 
 export async function importContent(version: VersionedReviewedPreprint): Promise<ImportContentOutput> {
-  const biorxivDoi = version.preprint.doi.startsWith('10.1101/');
-  if (!biorxivDoi) {
+  if (!version.preprint.content) {
```

Comment:
@E-Kemp Do you think we should still fail import content on a non-biorxiv version... until we have figured out the alternative storage for non-biorxiv content. Or, indeed, we currently don't list other non-biorxiv sources in docmaps yet, but if we did then it would attempt to fetch it as the import role or biorxiv role for devs (which has very little permissions beyond the biorxiv bucket). 

It just feels safer for now to say the whole code is built on the biorxiv bucket assumption, and complain loadly if not. 

/cc @nlisgo - FYI for tackling elife typeset (non biorxiv) preprints.

Reply:
I think that'll be a whole other ticket we need to think about, perhaps we need a fresh ticket to tackle the problem?

I'll remove that change in the mean time